tests/test-files/transform/Elm-0.18/Examples.formatted.elm

```diff
@@ -2,31 +2,15 @@ module Main exposing (bar, multilineList, ratio)
 
 
 ratio =
-    graphHeight
-        / (if range == 0 then
-            0.1
-
-           else
-            toFloat range
-          )
+    graphHeight / (if range == 0 then 0.1 else toFloat range)
 
 
 
 -- foo=(case x of {True->1;False->3})
 
 
 bar =
-    if
-        if a then
-            True
-
-        else
-            False
-    then
-        "a"
-
-    else
-        "b"
+    if if a then True else False then "a" else "b"
```

Comment:
It think this is the last remaining point for now.
Should we change the example, change the implementation, or let it as is?

Reply:
hmm, I guess it looks the original test was testing the indentation of the if statement in the predicate, so maybe the test input should change to have a single newline somewhere in the inner if statement.

Reply:
The original commit of the test from 2015 is:
> Handle multiline if conditions

So it seems to have been made nested to stress the implementation.
Anyway, it seems better to keep it multiline, so I changed it to:
```
bar = (if if a then
  True else False then "a" else "b")
```